### PR TITLE
[Matek H743] Add fix to resolve no startup of imu for matek_h743 target

### PR DESCRIPTION
**Pull request by matthewoots**
The current H743 target does not have the icm42688p driver which is needed to load the IMU for the latest `MATEK H743` boards

| H743 Wing V3 | H743 WLite |
| -- | -- |
| H743-WING-V3_1 | H743-WLITE_1 |

Added quick fix to resolve this and to load the driver with the correct rotation

Before fix:
```
nsh> dmesg
HW arch: MATEK_H743
FW git-hash: b45daac0e83cfab8c92262bdd4441fdcf46aee55
FW version: 1.13.0 0 (17629184)
FW git-branch: pr-matek_h743_add_v3
...
Board sensors: /etc/init.d/rc.board_sensors
ERROR [SPI_I2C] icm42605: no instance started (no device on bus?)
ERROR [SPI_I2C] icm20602: no instance started (no device on bus?)
ERROR [SPI_I2C] mpu6000: no instance started (no device on bus?)
dps310 #0 on I2C bus 2 address 0x76
qmc5883l #0 on I2C bus 1 (external) address 0xD
WARN  [SPI_I2C] Already running on bus 1
ekf2 [489:237]
...
```

After fix:
```
nsh> dmesg
HW arch: MATEK_H743
FW git-hash: b45daac0e83cfab8c92262bdd4441fdcf46aee55
FW version: 1.13.0 0 (17629184)
FW git-branch: pr-matek_h743_add_v3
...
Board sensors: /etc/init.d/rc.board_sensors
ERROR [SPI_I2C] icm42605: no instance started (no device on bus?)
ERROR [SPI_I2C] icm20602: no instance started (no device on bus?)
ERROR [SPI_I2C] mpu6000: no instance started (no device on bus?)
icm42688p #0 on SPI bus 1 rotation 12
dps310 #0 on I2C bus 2 address 0x76
qmc5883l #0 on I2C bus 1 (external) address 0xD
WARN  [SPI_I2C] Already running on bus 1
ekf2 [511:237]
...
```

### CODE REVIEW

**Comment by matthewoots** [comment]
If needed to show, here is the setup for the `Matek H743 V3`

photo_2022-09-09_22-35-00

File: boards/matek/h743/init/rc.board_sensors [hunk 1]
```diff
@@ -13,7 +13,11 @@ then
 fi
 
 # Internal SPI bus MPU-6000
-mpu6000 -R 12 -s start
+if ! mpu6000 -R 12 -s start
```
Comment (dagar) [inline boards/matek/h743/init/rc.board_sensors:16]: Add the specific bus (-b) while you're here?

Comment (matthewoots) [reply boards/matek/h743/init/rc.board_sensors]: Added it in [90adb9f](https://github.com/PX4/PX4-Autopilot/pull/20184/commits/90adb9fb6feca3ea48f3dbae94f8f1889e5a5aa4)

**Review by dagar — APPROVED** [review]

**Comment by dagar** [comment]
Not critical, but it might make sense to start the first IMU with `-q` to avoid the distracting error in the log. 

Ideally we'd have a more explicit mechanism to handle this situation, but I think it should really start with a mechanism to know which board version we're actually running on and then properly report failures for required sensors.

**Pull request merged**